### PR TITLE
fix(manager): correct warning typescript eslint

### PR DESCRIPTION
**Pull request by aboungnaseng-ovhcloud**
ref: MANAGER-14205

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-14205
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

Fix eslint warning for typescript

### CODE REVIEW

File: packages/manager/apps/catalog/src/utils/trackingError.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import { useShell } from '@ovh-ux/manager-react-core-application';
 import { useLocation } from 'react-router-dom';
-
 import { ErrorMessage, TRACKING_LABELS } from '@ovhcloud/manager-components';
-
-export interface ErrorObject {
-  [key: string]: any;
-}
+import { Environment } from '@ovh-ux/manager-config/dist/types';
+import { ErrorsObject } from '@/components/Error/Errors';
```
Comment (anooparveti) [inline packages/manager/apps/catalog/src/utils/trackingError.ts:5]:
```suggestion
import { ErrorObject } from '@/components/Error/Errors';
```

Comment (aboungnaseng-ovhcloud) [reply packages/manager/apps/catalog/src/utils/trackingError.ts:5]: 👍

File: packages/manager/apps/pci-rancher/src/components/Table/ProductStatusCell.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export default function ProductStatusCell({ cell }: Readonly<RancherCellData>) {
   };
 
   return label ? (
-    <OsdsChip inline color={colorByProductStatus[label]}>
+    <OsdsChip inline color={colorByProductStatus[label as RessourceStatus]}>
```
[the remark this answers is not in the document]
Comment (aboungnaseng-ovhcloud) [reply packages/manager/apps/pci-rancher/src/components/Table/ProductStatusCell.tsx:25]: 👍

**Review by vovh — APPROVED** [review]

File: packages/manager/apps/vrack-services/src/pages/RootWrapper.tsx [hunk 3]
```diff
@@ -18,7 +19,7 @@ export default function RootWrapper() {
   React.useEffect(() => {
     trackCurrentPage();
 
-    const currentPage = (matches[matches.length - 1]?.handle as any)
+    const currentPage = (matches[matches.length - 1]?.handle as RouteConfig)
```
Comment (chipp972) [inline packages/manager/apps/vrack-services/src/pages/RootWrapper.tsx:22]: handle is not a RouteConfig
It contains tracking and currentPage
```
type Handle = {
  tracking?: { pageType: PageType; pageName: PageName };
  currentPage?: string;
};
```

File: packages/manager/apps/vrack-services/playwright.config.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  timeout: 30 * 1000,
+  reporter: [['html', { open: 'on-failure' }]],
+  expect: {
+    timeout: 20000,
+  },
+  use: {
+    // Collect trace when retrying the failed test.
+    trace: 'retain-on-failure',
+  },
+});
```
Comment (chipp972) [inline packages/manager/apps/vrack-services/playwright.config.js:16]: why you change the extension ?

File: packages/manager/apps/vrack-services/src/pages/RootWrapper.tsx (hunk 3 above)
Comment (aboungnaseng-ovhcloud) [reply packages/manager/apps/vrack-services/src/pages/RootWrapper.tsx:22]: tracking is not merge yet

File: packages/manager/apps/vrack-services/playwright.config.js (hunk 4 above)
Comment (aboungnaseng-ovhcloud) [reply packages/manager/apps/vrack-services/playwright.config.js]: Its a mistake, It has been deleted

File: packages/manager/apps/vrack-services/src/pages/RootWrapper.tsx (hunk 3 above)
Comment (chipp972) [reply packages/manager/apps/vrack-services/src/pages/RootWrapper.tsx:22]: tracking on generator ?

**Review by chipp972 — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Pull request merged**